src/evtx_chunk.rs

```diff
@@ -53,26 +49,36 @@ impl Debug for EvtxChunkHeader {
 pub struct EvtxChunkData {
     pub header: EvtxChunkHeader,
     pub data: Vec<u8>,
+    pub string_cache: StringCache,
```

Comment:
So `EvtxChunkData` now owns the string_cache, but does not own the template cache?
The ownership module of the cache is brittle, we should have some notes regarding who owns what.